vnext/Microsoft.ReactNative/XamlUIService.idl

```diff
@@ -24,6 +24,12 @@ namespace Microsoft.ReactNative
     DOC_STRING("Dispatches an event to a JS component.")
     void DispatchEvent(XAML_NAMESPACE.FrameworkElement view, String eventName, JSValueArgWriter eventDataArgWriter);
 
+    DOC_STRING("Gets the root view for a given element.")
+    XAML_NAMESPACE.FrameworkElement GetReactRootElementForView(XAML_NAMESPACE.FrameworkElement view);
```

Comment:
does this get the `ReactRootView` element?
if so, it might make sense to:
- call this something like GetReactRootViewForElement, or omit the "ForElement" part
- link the doc to the doc for ReactRootView (you can do this with the `@ReactRootView` syntax)
- have it return ReactRootView instead of FrameworkElement